rdkit/__init__.py

```diff
@@ -7,17 +11,23 @@
   __version__ = 'Unknown'
   raise
 
-import logging
 logger = logging.getLogger("rdkit")
+
 # if we are running in a jupyter notebook, enable the extensions
 try:
   kernel_name = get_ipython().__class__.__name__
 
   if kernel_name == 'ZMQInteractiveShell':
+    logger.info("Enabling RDKit %s jupyter extensions" % __version__)
     from rdkit.Chem.Draw import IPythonConsole
-    from rdkit.Chem import LogWarningMsg, WrapLogs
-    WrapLogs()  
-    logger.info("Enabling RDKit %s jupyter extensions"%__version__)
-    
 except:
   pass
+
+# Do logging setup at the end, so users can suppress the
+# "enabling jupyter" message at the root logger.
+log_handler = logging.StreamHandler(sys.stderr)
+logger.addHandler(log_handler)
+logger.setLevel(logging.DEBUG)
+logger.propagate = False
```

Comment:
This makes the Python logger output all levels, to match the behavior of the C++ and Python `stderr` loggers.  It's a little weird to add a handler to a non-root logger like this, but I figured it was better than changing people's default logging setup whenever they `import rdkit`.